jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -202,6 +203,18 @@ public synchronized ResponseEntity<String> depositTim(String jsonString, Request
       String obfuscatedTimData = TimTransmogrifier.obfuscateRsuPassword(odeTimData.toJson());
       stringMsgProducer.send(odeProperties.getKafkaTopicOdeTimBroadcastJson(), null, obfuscatedTimData);
 
+      try {
+         String generatedBy = (new JSONObject(obfuscatedTimData)).getJSONObject("metadata").getString("recordGeneratedBy");
```

Comment:
Should we be using the object sent down to the topic.OdeTimJson below? Not sure what the difference between that object and this broadcast object is, but we _have_ been pulling from the topic.OdeTimJson so keeping with that would probably be best

Reply:
This now uses the object in the OdeTimJson topic directly